docs/usage.md

```diff
@@ -6,9 +6,9 @@ To do this, use the `generate` command. This command targets a specific dataset
 
 You can pass a local Fields generation configuration file.
 
-`go run main.go generate <package> <dataset> <version> --tot-size <quantity>`
+`go run main.go generate <package> <dataset> <version> --tot-events <quantity>`
 
-`package`, `dataset` and `version` are mandatory. `--tot-size` is mandatory.
+`package`, `dataset` and `version` are mandatory. `--tot-events` is not mandatory and in case it is not provided an infinite number of events will be generated.
```

Comment:
What do you think of setting default `tot-events` to `1`? I'm thinking about the principle of least surprise and helping with command discovery for new devs. An infinite stream would be a big surprise to me and not useful if I'm learning the tool or discovering its capabilities.
Creating 1 by default could also help when writing a template, when you generally want 1 event at a time for testing.